### DISCUSSION TITLE
Is there a way to set sort order in container chain

### DISCUSSION THREAD

**Discussion by happyHruska** [gerbera/gerbera#2746] [Q&A]
I am using docker version of Gerbera 1.12. When I import my music collection I see them listed, as expected, in Album/Aritsts/Genre/... containers. 
The sort order, as it looks like, is alphabetically according to album title. Is there a way to set sort order so that the albums listed under Artist container
are sorted by album year - so chronologically?

I was going over the documentation and the only think that I found is that if the upnpclass is set to UPNP_CLASS_CONTAINER_MUSIC_ALBUM then the items will be sorted by disk id and track number.

Thank you

**Comment by KarlStraussberger** [comment]
In general the sort order on a media player is as it is requested by the UPnP call. Default sort order is by title (most cases) / track number (albums).

The easiest way to change the sort order is to dig into scripting, evaluate properties, that are available in your music library (the way you set your tags) and set the title (what you see is what you get then) accordingly. This also ensures that media players that sort themselves know what you want the sort order to be.

For an introduction to scripting see https://docs.gerbera.io/en/stable/scripting.html.
